runtime/doc/eval.txt

```diff
@@ -1815,7 +1815,7 @@ getcmdpos()			Number	return cursor position in command-line
 getcmdtype()			String	return current command-line type
 getcmdwintype()			String	return current command-line window type
 getcurpos()			List	position of the cursor
-getcwd()			String	the current working directory
+getcwd( [{scope}])			String	the current working directory
```

Comment:
I think it's ok--the GitHub unified diff view just makes it look broken because of the leading `+`.

FWIW we try to use tabs in the old help docs to stay consistent.